lib/execution/executor.ex

```diff
@@ -894,4 +897,16 @@ defmodule WaspVM.Executor do
 
     Integer.undigits(zero_leading_map ++ bin, 2)
   end
+
+  defp create_entry(instruction) when is_tuple(instruction) == false, do: "#{instruction}"
```

Comment:
Same as above, don't compare against a boolean. We also don't need to interpolate into a string if we're not adding anything with it -- if we need it to be a string, we should call `Kernel.to_string\1`. 

This should be rewritten as 
```elixir
defp create_entry(instruction) when not is_tuple(instruction), do: instruction
```

Reply:
Fixed